tools/config/project.config.ts

```diff
@@ -31,6 +34,29 @@ export class ProjectConfig extends SeedConfig {
       // {src: `${this.CSS_SRC}/path-to-lib/test-lib.css`, inject: true, vendor: false},
     ];
 
+    // Add packages (e.g. lodash)
+    // this.ADDITIONAL_PACKAGES = [{
```

Comment:
Would you change it to a local variable instead of an instance one? We don't need `ADDITIONAL_PACKAGES` as part of the configuration but only as a container of information which is later intended to be transformed and produce a side-effect.